src/kernel/mocaml.ml

```diff
@@ -1,13 +1,37 @@
 open Std
+open Local_store.Compiler
 
+(* Instance of environment cache & btype unification log  *)
+
+type typer_state = Local_store.scope
+
+let bound = srefk None
```

Comment:
I think the code would be slightly easier to read if this was named `current_state`.